src/Utility/Inflector.php

```diff
@@ -643,8 +643,12 @@ public static function dasherize($string)
      */
     public static function humanize($string, $delimiter = '_')
```

Comment:
If you accept array, why not changing the doc block?

Reply:
Sorry wrong doc block